packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -144,10 +145,12 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
   void _initChartController() {
     final desc = Description()..enabled = false;
     _chartController = BarChartController(
+      backgroundColor: ThemedColor(Colors.white, Colors.grey[850]),
```

Comment:
Add a TODO to make this a general background color constant if it isn't already.

Reply:
done